deploy/templates/service-template.yml

```diff
@@ -497,8 +492,6 @@ objects:
                   value: ${MIGRATION_PLANNER_UI_URL}/migrate/wizard
                 - name: MIGRATION_PLANNER_IMAGE_URL
                   value: ${MIGRATION_PLANNER_IMAGE_URL}
-                - name: MIGRATION_PLANNER_AGENT_IMAGE
```

Comment:
I think we need to keep this environment variable, because you keep using the image from the registry
This might be different image for staging and production